docs/main/faqs/faqs.md

```diff
@@ -664,6 +664,12 @@ It is recommended that Replicated PV Mayastor volumes be provisioned with a mini
 
 [Go to top](#top)
 
+### Should I install the Replicated PV Mayastor/Mayastor Helm chart alongside the OpenEBS/OpenEBS Helm release?
```

Comment:
This is not the title name of the storage engines as we see in most places, the helm chart names are `mayastor/mayastor` and `openebs/openebs`.
```suggestion
### Should I install the mayastor/mayastor Helm chart alongside an openebs/openebs Helm release?
```